### DISCUSSION TITLE
What happens when a calculated checksum is a mismatch ?

### DISCUSSION THREAD

**Discussion by redmoon1945** [artemvlas/veretino#2] [General]
It means either that : 
* the file is corrupted
* the original file has been renamed (and maybe moved into another subdir) and a new file is using the old name
How veretino copes with these cases ?

**Comment by artemvlas** [comment]
A Mismatched checksum means that the current file data is different from what it was at the time the checksum was added/updated: the file was changed or damaged, any change in at least one bit of its data.
If the file has been renamed, it will be marked as New (there will also be a Missing item). In this case, you need to update the database (add New ones, clear Missing ones).
